website/project/model.py

```diff
@@ -4000,6 +4003,17 @@ def _on_reject(self, user):
             auth=Auth(user),
         )
 
+class AlternativeCitation(StoredObject):
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+    name = fields.StringField(required=True)
```

Comment:
There should maxlength validators on `name` and `text`. I think 255 and 2048 would be fine.